web_src/js/utils/dom.ts

```diff
@@ -302,8 +302,15 @@ export function replaceTextareaSelection(textarea: HTMLTextAreaElement, text: st
 
 // Warning: Do not enter any unsanitized variables here
 export function createElementFromHTML(htmlString: string): HTMLElement {
+  htmlString = htmlString.trim();
+  // some tags like "tr" are special, it must use a correct parent container to create
+  if (htmlString.startsWith('<tr')) {
```

Comment:
```suggestion
  if (htmlString.toLowerCase().startsWith('<tr')) {
```

Reply:
I do not think we should accept such dirty HTML tag in code.